modules/ROOT/pages/design-scenarios-s6m3-for-published-apis.adoc

```diff
@@ -0,0 +1,19 @@
+= Using Published RAML API Specifications in Studio 6._x_ and Mule Runtime 3._x_
+
+As described in xref::design-modify-raml-specs-conform.adoc[Modify Published RAML API Specifications to Conform Completely to RAML 0.8 or 1.0], the text editor in Design Center is now (as of January 10, 2019) more precise in ensuring that the RAML in your API specifications is valid. Although RAML API specifications that you have already published might contain invalid RAML, you will have a long grace period in which to make corrections.
+
+The information below explains how Anypoint Studio 6._x_ and Mule Runtime 3._x_ interact with RAML API specifications that are published both during and after the grace period.
+
+== Scenarios involving Anypoint Studio 6._x_ and Mule Runtime 3._x_
+
+* Both during and after the grace period, in the majority of cases, a valid RAML API specification that is published to Exchange from the text editor will not be flagged with additional RAML errors when opened in Studio 6. Moreover, the implementation that is created in Studio 6 will be able to run without errors in Mule Runtime 3.
+* During the grace period, it is possible in some cases to publish to Exchange an API specification that still has invalid RAML according to the text editor, yet open the specification in Studio 6 and see no invalid RAML flagged. In such cases, there is in fact invalid RAML in the specification, but Studio 6 is not flagging it. Also, the implementation that is created in Studio 6 will be able to run without errors in Mule Runtime 3.
```

Comment:
Remove "in fact" - it's doesn't add anything.

Reply:
I simply need to revert. Next Wednesday, I will revert the revert.